addons/web/static/tests/views/kanban/kanban_view.test.js

```diff
@@ -6292,6 +6292,46 @@ test("quick create column with x_name as _rec_name", async () => {
     expect(".o_kanban_group").toHaveCount(3, { message: "should now have three columns" });
 });
 
+test.tags("desktop")("count should be hidden in kanban folded stage if zero", async () => {
```

Comment:
you already assert this (indirectly) with the change you had to make in the other test. However, what you don't test is that you don't display the count if we are in sample mode. Could you replace this test by a test asserting that?

The test should be short:
```js
// on need an onRpc to mock the web_read_group, to return empty groups
// e.g. like in test "click on empty kanban must shake the NEW button"

await mountView({
        type: "kanban",
        resModel: "partner",
        arch: `
            <kanban sample="1">
                <templates>
                    <t t-name="kanban-card">
                        <field name="foo"/>
                    </t>
                </templates>
            </kanban>`,
        groupBy: ["some_many2one_field"],
        domain: [["id", "<", 0]],
    });

    // assert you're in sample mode
    // assert that there's a folded column
    // assert the count isn't displayed
```

Reply:
I have made the required changes.